src/sass/layout/scanning-page/_hero.scss

```diff
@@ -0,0 +1,84 @@
+
+.scanning-page {
+    padding-top: 11px;
+    @media screen and (min-width: 320px) and (max-width: 767px) {
+        padding-bottom: 247px;
+    }
+    
+    background-image: url(../images/scanning-page/scanning_photo_mobile.jpg);
+    background-repeat: no-repeat;
+    background-position: bottom;
+}
+.scanning-page__list {
+    @extend %list;
+}
+.scanning-page__desc {
+@include font(16px, 300, 20px, 0.03em);
+margin-bottom: 16px;
+color: #2B2B2B;
+width: 280px;
+}
+
+.last-desc {
+    margin-bottom: 0px;
+}
+
+.scanning-page__title {
+@include font(24px, 700, 30px, 0.03em);
+margin-bottom: 16px;
+color: #487996;
+text-transform: uppercase;
+}
+
+@media screen and (min-width: 768px) and (max-width: 1443px) {
+    .scanning-page__desc {
+        width: 515px;
+       &:not(:last-child) {
+            margin-bottom: 16px;
+        }
+    }
+    .scanning-page__title {
+        font-size: 36px;
+        }
+    
+    .last-desc {
+      margin-bottom: 0px;
+    }    
+        .scanning-page {
+            padding-top: 23px;
+            padding-bottom: 52px;
+            background-image: url(../images/scanning-page/scanning_photo_tablet.jpg);
+            background-position: right bottom 77px;
+        }    
+}
+
+
+@media screen and (min-width: 1444px) {
+    .scanning-page__block {
+        display: flex;
+    }
+       
+.scanning-page__desc {
+    font-size: 18px;
+    width: 515px;
+   
+}
+.scanning-page__title {
+    font-size: 48px;
+    margin-bottom: 24px;
+}
+.scanning-page {
```

Comment:
you can put bem element inside bem block like 

.scanning-page {
    padding-top: 24px;
    padding-bottom: 100px;
    background-image: url(../images/scanning-page/scanning_photo_desktop.jpg);
    background-position: right 189px bottom;
         &__title {
                 font-size: 48px;
                 margin-bottom: 24px;
             }
         &__desc {
                  font-size: 18px;
                  width: 515px;
               }    
}